src/main/java/com/infoshareacademy/controller/RecipeController.java

```diff
@@ -105,4 +105,10 @@ public String deleteRecipe(@PathVariable Long id) {
         recipeService.deleteRecipeById(id);
         return "redirect:/recipes";
     }
+
+    @GetMapping("/deleteAll")
```

Comment:
https://nordicapis.com/10-best-practices-for-naming-api-endpoints/

Poprawiła bym nazwę end-pointa wg.tego, co Bartek zamieszczał